src/SignalR/clients/ts/signalr-protocol-msgpack/src/MessagePackHubProtocol.ts

```diff
@@ -32,6 +33,8 @@ export class MessagePackHubProtocol implements IHubProtocol {
     private readonly voidResult = 2;
     private readonly nonVoidResult = 3;
 
+    constructor(private messagePackOptions?: MessagePackOptions) { }
```

Comment:
Hrm, I'd kinda prefer to either a) use a type from msgpack5's typings or b) use `any` since if a user changes the version of msgpack5 they use, and that version brings new options they would get TypeScript errors trying to use them.

Reply:
That is a good point. I have updated to use `any`.